doc/api/errors.md

```diff
@@ -189,7 +189,9 @@ intercepted.
 A generic JavaScript `Error` object that does not denote any specific
 circumstance of why the error occurred. `Error` objects capture a "stack trace"
 detailing the point in the code at which the `Error` was instantiated, and may
-provide a text description of the error.
+provide a text description of the error. For crypto only, `Error` objects will
+include the OpenSSL error stack, if it's available when the error is thrown, in
+a separate property.
```

Comment:
tiny nits: use long forms (`it's` → `it is`) and I’d put the sentence in a new paragraph.